examples/vae_renyi.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+"""Variational auto-encoder for MNIST data using Renyi variational objective
+[@li2016renyi]
+
+#### Notes
+This example is almost exactly similar to example/vae.py.
```

Comment:
Sorry for the miscommunication. What I meant was that you can edit `vae.py`, comment out the 1-2 lines of code to use `ed.RenyiDivergence`, and add these notes there. This helps to compress the content in the examples, c.f., https://github.com/blei-lab/edward/blob/master/examples/bayesian_logistic_regression.py#L51.

Reply:
Removed vae_renyi.py and modifed vae.py instead.
The version of vae.py I had wasn't running though. So I've modified it quite a bit.

Reply:
Are you using the latest version of Edward? We updated a few details in vae.py so it actually runs better. For example, you should be using the observations library and a generator, which is far more transparent than the `mnist_data` class from that TensorFlow tutorial.

In addition, since `vae.py` is also our canonical VAE example, I prefer keeping it as `ed.KLqp` as the default, and with the renyi divergence option commented out; similarly, the top-level comments should be written in-line near the renyi divergence option instead.

If you have thoughts otherwise, happy to take alternative suggestions.